src/server.h

```diff
@@ -1419,6 +1419,7 @@ struct redisServer {
                                       REDISMODULE_CLUSTER_FLAG_*. */
     int cluster_allow_reads_when_down; /* Are reads allowed when the cluster
                                         is down? */
+    int cluster_config_flock_fd;       /* cluster config fd, will be flock */
```

Comment:
Can we call this cluster_config_file_lock_fd instead? Flock is the name of the API, it seems weird to use it as an adjective. Also, initialize this to -1 so we don't try to close stdin.

Reply:
fixed, Thanks